kfs-core/src/main/java/edu/arizona/kfs/fp/businessobject/lookup/UaDisbursementPayeeLookupableHelperServiceImpl.java

```diff
@@ -65,15 +71,47 @@
 	protected List<DisbursementPayee> getPersonAsPayeesByEmployeeStatusCode(Map<String, String> fieldValues, String employeeStatusCode) {
 		List<DisbursementPayee> payeeList = new ArrayList<DisbursementPayee>();
 
-		fieldValues.put(KIMPropertyConstants.Person.EMPLOYEE_STATUS_CODE, employeeStatusCode);
-		List<? extends Person> persons = KimApiServiceLocator.getPersonService().findPeople(fieldValues);
+		Map<String, String> personFieldValues = this.getPersonFieldValues(fieldValues);
+		personFieldValues.put(KIMPropertyConstants.Person.EMPLOYEE_STATUS_CODE, employeeStatusCode);
+		List<? extends Person> persons = getPersonService().findPeople(personFieldValues);
 
 		for (Person personDetail : persons) {
-			DisbursementPayee payee = getPayeeFromPerson(personDetail, fieldValues);
+			DisbursementPayee payee = getPayeeFromPerson(personDetail, personFieldValues);
 			payeeList.add(payee);
 		}
 
 		return payeeList;
 	}
 
+	private static volatile PersonService personService;

```

Comment:
KualiCo's standard had been to use "private static volatile" for services, and hadn't been told to do otherwise. This can be moved to the top, forgot to do that when I was cleaning up the code.

Looking into what volatile does in Java, every thread that accesses the static field uses the current values, rather than a (potentially) cached value which could cause problems.